src/shared/components/create/NoteCreator.tsx

```diff
@@ -207,16 +212,7 @@ function NoteCreator({handleClose, quotedEvent, repliedEvent}: NoteCreatorProps)
     event.kind = 1
     event.content = noteContent
     event.tags = []
-    if (repliedEvent) {
-      event.tags = [
-        ["p", repliedEvent.pubkey],
-        ["e", repliedEvent.id, "", "reply", repliedEvent.pubkey],
-      ]
-    }
-    if (quotedEvent) {
-      event.tags.push(["e", quotedEvent.id, "", "mention", quotedEvent.pubkey])
-    }
```

Comment:
Redundant because the add tags function was already doing all of this. Not sure why it's needed again.